pkgs/top-level/python-packages.nix

```diff
@@ -13837,6 +13853,7 @@ in {
     };
   };
 
+  magic-wormhole = callPackage ../tools/misc/magic-wormhole {};
```

Comment:
@FRidh I was following the example of `youtube-dl`, which is there. What's the difference?